README.rst

```diff
@@ -8,7 +8,18 @@ MantisShrimp
 
 --------------
 
-``MantisShrimp`` provides a unified data API for object detection that can be used with any library (it can even be injected in code that was not supposed to work as a library, take a look at ``hub/detr`` for an example).
+.. image:: images/mantisshrimp-logo.png
+
+Why Mantishrimp
+---------------
+- Mantisshrimp: An object-detection library
+- Built on top of different libraries/framework such as Pytorch Lightning and Pytorch
```

Comment:
It's not really built on top of those libraries, rather it coexists with them. I'm not sure how to phrase it correctly, but the pluggable idea sounded good

Reply:
We will rephrase it:

1st attempt:
Bridge different libraries/framework such as Pytorch Lightning and Pytorch

Reply:
You can put fastai in there ^^

Reply:
2nd attempt:
Connects to different libraries/framework such as Pytorch Lightning and Pytorch

Reply:
I think 2nd attempt is easier to understand

Reply:
Connects to different libraries/framework such as fastai, Pytorch Lightning, and Pytorch